app/gui/src/project-view/components/ComponentBrowser/ComponentEditor.vue

```diff
@@ -76,8 +76,8 @@ const rootStyle = computed(() => {
     </div>
     <div class="componentEditorContent">
       <CodeMirrorRoot ref="editorRoot" class="componentEditorInput" />
-      <div v-if="props.mode.mode === 'componentBrowsing'" class="componentEditorLabel">
-        <ComponentEditorLabel
+      <div v-if="props.mode.mode === 'componentBrowsing'" class="componentTypeLabel">
```

Comment:
I think it's confusing to apply the `componentTypeLabel` class here, when there's a component of the same name (with the same class)--I'd use a distinct class name here, with two separate `margin: 0` CSS declarations for the two classes if they both need it.

Reply:
Fixed